Dockerfile.template

```diff
@@ -1,15 +1,17 @@
 # base-image for node on any machine using a template variable,
 # see more about dockerfile templates here:http://docs.resin.io/pages/deployment/docker-templates
 # Note the node:slim image doesn't have node-gyp
-FROM resin/%%RESIN_MACHINE_NAME%%-node:6
+FROM resin/%%RESIN_MACHINE_NAME%%-alpine-node:slim
```

Comment:
This should be 6-slim @curcuz, otherwise it'll track the latest node version and potentially break on new major node versions